.github/workflows/avm.res.azure-stack-hci.cluster.yml

```diff
@@ -0,0 +1,90 @@
+name: "avm.res.azure-stack-hci.cluster"
+
+on:
+  schedule:
+    - cron: "0 12 1/15 * *" # Bi-Weekly Test (on 1st & 15th of month)
+  workflow_dispatch:
+    inputs:
+      staticValidation:
+        type: boolean
+        description: "Execute static validation"
+        required: false
+        default: true
+      deploymentValidation:
+        type: boolean
+        description: "Execute deployment validation"
+        required: false
+        default: true
+      removeDeployment:
+        type: boolean
+        description: "Remove deployed module"
+        required: false
+        default: true
+      customLocation:
+        type: string
+        description: "Default location overwrite (e.g., eastus)"
+        required: false
+  push:
+    branches:
+      - main
+    paths:
+      - ".github/actions/templates/avm-**"
+      - ".github/workflows/avm.template.module.yml"
+      - ".github/workflows/avm.res.azure-stack-hci.cluster.yml"
+      - "avm/res/api-management/service/**"
```

Comment:
Outdated

Reply:
Updated workflow from template at https://azure.github.io/Azure-Verified-Modules/contributing/bicep/bicep-contribution-flow/#4-implement-your-contribution

Reply:
Thanks. I think the template overall was fine, but as you can see in the referenced line of the comment, it was referencing the API management module 😉